### ISSUE TITLE
Add option to have language code, but no flag

### ISSUE THREAD

**Issue by tuxedo0801** [WPGlobus/WPGlobus#46]
In WPGLobus config, I can choose between differen language selector modes.

There's one missing:

"Language code WITHOUT flag (de, ru, it, ...)"

Choosing complete language name wihtout flag looks like it's a common menu-entry and not a kind of language chooser.

Or go for country/language-independant icon. See: http://www.languageicon.org/

But all in all: Flags do not reflect a language at all.

Please think about if for new next release.

**Comment by tivnet** [comment]
@tuxedo0801 Yes, of course, language and flag are not the same. As a Canadian, I know that very well :)

"Language without flag" is solved easily with a CSS rule hiding flags.

Icons are in the "flags" folder. We started with simple two-letter codes there but recently added full locale names. Will continue with that.

The prefix itself is also flexible. I can use /ca/ for English Canada and /qc/ for French Canada (Quebec).

And "all in all" - the language selector is a tiny part of WPGlobus. You can write any replacement for it, and publish as an open-source add-on. The World will thank you!

**Comment by tuxedo0801** [comment]
> "Language without flag" is solved easily with a CSS rule hiding flags.

I'm not very familar with CSS. Could you give me a hint what to add where?

**Comment by alexgff** [comment]
@tuxedo0801 
What is the URL of your site?

**Comment by tuxedo0801** [comment]
htp://www.konnekting.de

**Comment by alexgff** [comment]
@tuxedo0801 
On page yoursite/wp-admin/admin.php?page=wpglobus_options&tab=1
add to Custom JS Code and save 

jQuery(document).ready(function ($) {
    $('.primary-menu  .wpglobus-selector-link .wpglobus_flag_de').css({'padding':0}).removeClass('wpglobus_flag_de');
    $('.primary-menu  .wpglobus-selector-link .wpglobus_flag_en').css({'padding':0}).removeClass('wpglobus_flag_en');
});

**Comment by tuxedo0801** [comment]
thanks a lot. works like a charm.

**Comment by ivanlaharnar** [comment]
Just add in the field CUSTOM CSS this code.
.wpglobus_flag_en{
background-size: 0 0;
padding: 0;
}
That's for language english, if you have any other language change class name.

Regards, Ivan
